### DISCUSSION TITLE
Sound Volume

### DISCUSSION THREAD

**Discussion by nilsn** [kaiwinter/nfcsonos#2] [Ideas]
Hi Kai,

awesome app, thank you! 
I'm really looking forward to the volume preset. Any idea when to have it live?

I have this functionality in a similar setup to play Sonos favorites and I love it, NFC would be even better: I use a Hue switch which sends a zigbee command to a Raspberry Pi with a zigbee dongle that runs home assistant. The HA OS triggers a webhook command on IFTTT and there I can set the Fav, sound level, groups etc. 

Maybou you can add a "send webhook" functionality to your app as an additional alternative to control Sonos. 
It works on Android out of the box, but then it always opens a new chrome tab.

Thx! 
Best, Nils

**Comment by kaiwinter** [comment]
The volume preset was just an idea but I had no real use for it yet. I think one global preset would be sufficient and easy to add to the settings tab. What do you think? 

What you describe about the webhook sounds interesting. I can imagine many useful scenarios in combination with IFTTT. But I don’t see this in the NFC Sonos Controller app. That should better be an own app which can be based on this one. And there are already apps like MacroDroid why don't you use those to connect a NFC sticker to a webhook?
